### PR TITLE
Deploy messaging only

### PR DESCRIPTION
**Pull request by LayneHaber**
## Description

Updates the init script to:
- optionally initialize the liquidity layer
- optionally send init transactions via multi send (in a single transaction)

Current status:
- Has utilities in place to upgrade using ^^
- Has command line options to specify the type of init

Remaining:
- [ ] Testing
- [ ] Update `setupMessaging`, top-level `init` to use multisend
- [ ] (optional) Implement only liquidity layer upgrading

## Type of change

- [ ] Docs change / dependency upgrade
- [x] Configuration / tooling changes
- [x] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/deployments/contracts/src/init/init.ts [hunk 1]
```diff
@@ -53,6 +57,21 @@ export const sanitizeAndInit = async () => {
...
+  if (typeof liquidity !== "boolean") {
+    throw new Error(`"liquidity" option must be a boolean, liquidity: ${liquidity}`);
+  }
+
+  if (!messaging) {
+    throw new Error(
+      `"messaging" option must be true, messaging: ${messaging}. (can be false in the future, but this script should be updated)`,
+    );
+  }
+  // TODO: when messaging
```
Comment (jakekidd) [inline packages/deployments/contracts/src/init/init.ts:69]: ?

File: packages/deployments/contracts/src/init/helpers/tx.ts [hunk 2]
```diff
@@ -78,9 +101,90 @@ export const updateIfNeeded = async <T>(schema: CallSchema<T>): Promise<void> =>
...
+  return {
+    checkResult:
+      desired != undefined
+        ? {
+            method: readCall,
+            desired,
+          }
+        : undefined,
+    data: getMultisendTransaction(
+      "delegatecall",
```
Comment (jakekidd) [inline packages/deployments/contracts/src/init/helpers/tx.ts:163]: shouldnt it be a call if we're updating values?

File: packages/deployments/contracts/src/init/init.ts (hunk 1 above)
Comment (LayneHaber) [reply packages/deployments/contracts/src/init/init.ts:69]: honestly idk -- update when we want to init only liquidity layer is the guess?

**Comment by LayneHaber** [comment]
closing

**Pull request closed without merging**